From 9eefaf0498e0afaf1144ecdaac1b9c6c2d8a6ec7 Mon Sep 17 00:00:00 2001
From: TungDang <tung.dang.jy@rvc.renesas.com>
Date: Wed, 28 Dec 2016 10:59:48 +0700
Subject: [PATCH] Fix binding loop for declarative-camera

qrc:///VideoCaptureControls.qml:112: Error: Cannot assign [undefined] to QString
qrc:///CameraListButton.qml:65:5: QML CameraListPopup: Binding loop detected for property "currentValue"

Signed-off-by: TungDang <tung.dang.jy@rvc.renesas.com>
---
 examples/multimedia/declarative-camera/CameraListButton.qml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/examples/multimedia/declarative-camera/CameraListButton.qml b/examples/multimedia/declarative-camera/CameraListButton.qml
index 892bb4e..b53b57a 100644
--- a/examples/multimedia/declarative-camera/CameraListButton.qml
+++ b/examples/multimedia/declarative-camera/CameraListButton.qml
@@ -69,8 +69,6 @@ Item {
         anchors.top: parent.top
         visible: opacity > 0

-        currentValue: cameraListButton.value
-
         onSelected: popup.toggle()
     }
 }
--
1.9.1
